### DISCUSSION TITLE
Any consideration on selecting LLM

### DISCUSSION THREAD

**Discussion by heylamourding** [haotian-liu/LLaVA#152] [General]
Hi guys,

Thanks for the great work! 

I notice you also support MPT-7B-Chat. May I know what's the reason to select MPT-7B-Chat from MPT series?

**Comment by haotian-liu** [comment]
Hi @heylamourding, thank you for your interest in our work.  We choose MPT-7B-Chat because its pretraining dataset has the most similar distribution as to Vicuna, thus we are expecting a similar behavior, and also a larger likelihood that our findings hold on this new model as well.

We are planning to see the performance of other variants as well, but it is not our top priority at this point.  Thanks!

**Comment by heylamourding** [comment]
Hi @haotian-liu  thanks for the explanation. Is multi-turn chat-based LLM the pre-requisite?

**Comment by heylamourding** [comment]
Could you help suggest: what's the minimum requirement of LLM for enabling the power of LLAVA?

**Comment by haotian-liu** [comment]
If you use the LLaVA tuned with single-turn LLM with single-turn QA, I believe it will also work well.  However, it may not work as well (it may work okay) if you ask follow-up questions, similar to the findings in LLMs.
